lib/model/QubitInformationObject.php

```diff
@@ -2626,28 +2631,18 @@ public function importPhysicalObject($location, $name = false, $type = false, $l
       : false;
 
     // Check for an existing physical object within this collection with the same name
-    if ($this->parentId)
+    if ($options['collectionId'])
```

Comment:
I think it will throw a PHP Notice if 'collectionId' doesn't exist, so maybe wrapping it with !empty(...) is best? If 'collectionId' should be there always though, feel free to disregard.